specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json

```diff
@@ -975,6 +1028,120 @@
     }
   },
   "definitions": {
+    "KeyReleaseCondition": {
+      "properties": {
+        "claim": {
+          "x-ms-client-name": "claim name",
+          "type": "string",
+          "minLength": 1,
+          "description": "claim name"
+        },
+        "condition": {
+          "x-ms-client-name": "condition to test",
+          "type": "string",
+          "minLength": 1,
+          "description": "condition to test",
+          "enum": [
+            "equals"
+          ],
+          "x-ms-enum": {
+            "name": "KeyReleaseCondition",
```

Comment:
You can't have the same enum name as the class name that contains it. That will cause an error during generation.